components/SelectorBase/SelectorBase.content.comp.cy.js

```diff
@@ -144,7 +144,7 @@ describe('Test the default SelectorBase content', () => {
         label: `TheLabel`,
         options: ['One', 'Two', 'Three', 'Four', 'Five'],
         onReady: readySpy,
-        includeAddButton: true,
+        popupUrl: '',
```

Comment:
The empty string (`''`) is kind of special, so passing it here makes it seem at first look like you are doing a test specifically about the empty string.  But it looks like the popup appears as long as `popupUrl` is not `null` - if I am reading things correctly.  So maybe use a non-`null` value here that is not the `''`, just to make it more clear to the reader that you are providing a value here and not specifically testing the behavior of the `''`,